### PR TITLE
Modified the usage of IoT Setup Manager

### PR DESCRIPTION
**Pull request by jpark0123**
We clearly commented that only the USB SD card reader is supported.

Signed-off-by: Jihoon Park <jpark0@samsung.com>

### Change Description ###

 - Clearly commented that only the USB SD card reader is supported in IoT Setup Manager

### CODE REVIEW

File: docs/iot/getting-started/tizen-image-download-flash.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ The IoT Setup Manager view contains the following fields:
 
 -   **Select Drive**
```
Comment (sabeena21) [inline docs/iot/getting-started/tizen-image-download-flash.md]: Please change the following:
Line 59 – Please change “Windows 7 and above (64 and 32 bit)” to “Windows 7 and earlier (64-bit and 32-bit)”
Line 60 – Please change “Ubuntu 14.04 LTS and above (64 and 32 bit)” to “Ubuntu 14.04 LTS and earlier (64-bit and 32-bit)”
Line 64 – Please change “Java Runtime Environment (JRE) version 1.8 or above” to “Java Runtime Environment (JRE) version 1.8 or earlier”

File: docs/iot/getting-started/tizen-image-download-flash.md [hunk 2]
```diff
@@ -232,7 +232,8 @@ To flash Tizen on your SD card using the IoT Setup Manager:
 
         > **Note**
         >
-        > If the Select Drive does not show anything, detach and insert the SD card again into the computer.
+        > - Currently, the USB SD card reader is only supported, but the internal PC SD card slot is not supported yet.
```
Comment (sabeena21) [inline docs/iot/getting-started/tizen-image-download-flash.md]: Please rephrase as following:
Currently, the USB SD card reader is only supported. The internal PC SD slot is not supported.

File: docs/iot/getting-started/tizen-image-download-flash.md [hunk 3]
```diff
@@ -232,7 +232,8 @@ To flash Tizen on your SD card using the IoT Setup Manager:
...
+        > - If the Select Drive does not show anything, detach and insert the SD card again into the computer.
```
Comment (sabeena21) [inline docs/iot/getting-started/tizen-image-download-flash.md]: Please change as:
Also, if the **Select Drive** does not show anything, detach and insert the SD card again into the computer.

File: docs/iot/getting-started/tizen-image-download-flash.md (hunk 1 above)
Comment (jpark0123) [reply docs/iot/getting-started/tizen-image-download-flash.md]: I think it should be “Windows 7 and later (64 and 32 bit)”, “Ubuntu 14.04 LTS and later (64-bit and 32-bit)” and  “Java Runtime Environment (JRE) version 1.8 or later”

Comment (sabeena21) [reply docs/iot/getting-started/tizen-image-download-flash.md]: @jpark0123, it is correct to use later.

**Comment by mijin85cho** [comment]
Thanks for the check, sabeena21.

**Comment by mijin85cho** [comment]
TD Sync Completed.

**Pull request merged**